draft-ietf-tls-tls13.md

```diff
@@ -2782,8 +2805,8 @@ The following rules apply to the certificates sent by the server:
                        in the server's KeyShare extension.
                        Note: ECDHE_RSA is defined in [RFC4492].
 
-    ECDHE_ECDSA        ECDSA-capable public key; the certificate MUST
-                       allow the key to be used for signing with the
+    ECDHE_ECDSA        ECDSA- or EdDSA-capable public key; the certificate
```

Comment:
just drop the dashes -> "ECDSA or EdDSA capable public key;"
probably could drop the "capable" too -> "ECDSA or EdDSA public key;"